### DISCUSSION TITLE
Is it possible to mount an NTFS volume case insensitive?

### DISCUSSION THREAD

**Discussion by mischw** [fox-it/dissect.target#765] [Q&A] [answered]
NTFS volumes are typically case insensitive. When mounted with target-mount on Linux I saw that it is actually case sensitive. Some forensic tools rely on the fact that a file can be found in its upper and lower case spelling. To make my life easier I thought it might be the best to mount the volume case insensitive. While target-mount supports an `--options` argument, I did not find anything related to case sensitivity with fuse.

Is there a way to accomplish this as of now or are there plans to implement such feature?

**Comment by Schamper** [comment, marked as answer]
Can you provide some more information? For me, `target-mount` already behaves case-insensitive on Windows systems:

```bash
$ ls
DLLs  include  libs         Microsoft.VC90.CRT.manifest  NEWS.txt      python.exe   README.txt  tcl
Doc   Lib      LICENSE.txt  msvcr90.dll                  python27.dll  pythonw.exe  Scripts     Tools

$ cd TOOLS

$ ls
i18n  pynche  Scripts  versioncheck  webchecker
```

**Reply by mischw** [reply]
I think you are right, sorry. I was fooled when using the EZ `rla` tool.

Example with mixed case names:
```bash
ls
[...]
NTUSER.DAT
ntuser.dat.LOG1
ntuser.dat.LOG2
[...]
```

It works when using `rla` on the lower case name `ntuser.dat`, even though the file is actually upper case:
```bash
dotnet rla.dll -f ntuser.dat --out [...]
Processing hive ntuser.dat
[...]
Replaying log file: ./ntuser.dat.LOG1
```

But it does not find the `*.LOG` files when using the upper case name `NTUSER.DAT`:
```bash
dotnet rla.dll -f NTUSER.DAT --out [...]
Processing hive NTUSER.DAT
[...]
Hive NTUSER.DAT is dirty, but no logs were found in the same directory.
```

`rla` probably still does some case sensitive string comparison internally. Nothing wrong with dissect :)
